x-pack/plugins/uptime/public/components/common/header/action_menu_content.test.tsx

```diff
@@ -7,9 +7,23 @@
 
 import React from 'react';
 import { fireEvent, waitFor } from '@testing-library/react';
+// We are using this inside a `jest.mock` call. Jest requires dynamic dependencies to be prefixed with `mock`
+import { coreMock as mockCoreMock } from 'src/core/public/mocks';
 import { render } from '../../../lib/helper/rtl_helpers';
 import { ActionMenuContent } from './action_menu_content';
 
+const mockStartServices = mockCoreMock.createStart();
+jest.mock('../../../hooks/use_kibana', () => ({
+  useKibanaContextForPlugin: () => ({
+    services: {
+      ...mockStartServices,
+      observability: {
+        useRulesLink: jest.fn().mockImplementation(() => ({ href: 'newRuleLink' })),
+      },
+    },
+  }),
+}));
```

Comment:
same suggestion, move it to share test helper in `../../../lib/helper/rtl_helpers`